index.html

```diff
@@ -1,69 +1,608 @@
+	<script src="./js/main.js" defer></script>
+</head>
 
-	<link rel="preconnect" href="https://fonts.googleapis.com">
-	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-	<link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
+<body class="font-inter antialiased bg-zinc-900 text-zinc-200 tracking-tight">
 
-	<script type="module" src="./static/js/countUp.min.js"></script>
-	<script type="module" src="./static/js/script.js"></script>
+	<!-- Page wrapper -->
+	<div class="flex flex-col min-h-screen overflow-hidden">
 
-  	<link rel="stylesheet" href="./static/css/styles.css">
-	<!-- <link rel="preload" href="https://via.placeholder.com/160" as="image"> -->
-  <!-- <script src="https://kit.fontawesome.com/efd3f62588.js" crossorigin="anonymous"></script> -->
+		<!-- Site header -->
+		<header class="absolute w-full z-30">
+			<div class="max-w-6xl mx-auto px-4 sm:px-6">
+				<div class="flex items-center justify-between h-20">
 
-</head>
-<body>
-	<header>
-		<div class="title-section">
-			<h1 id="page-header">Kal<span class="emph">vis</span> Studio</h1>
-			<h3 id="page-subheader">Own your <span class="emph">future</span></h3>
-			<img src="./static/assets/img/headers/warehouse.webp" alt="Warehouse with 18th century interior walls flooded by light from all directions. In the center, are futuristic screens illuminating their surroundings in blue light." />
-		</div>
-	</header>
-	<nav class="flex primary-navigation">
-
-		<a onClick="route()" href="/home">
-			<img data-active="true" src="./static/assets/img/logos/logo_kalvis_no_bg.png" alt="Kalvis logo: Two orange swung shapes forming a bird with 'Kalvis' spelled out next to it"/>
-		</a>
-
-		<button class="mobile-nav-toggle" aria-controls="primary-navigation" aria-expanded="false">
-			<span class="sr-only">Menu</span>
-		</button>
-
-		<ul id="primary-navigation" class="flex" data-visible="false">
-			<li class="nav-item" data-active="true"><a onClick="route()" href="/home">Home</a></li>
-			<li class="nav-item" data-active="false"><a onClick="route()" href="/about">About</a></li>
-			<li class="nav-item" data-active="false"><a onClick="route()" href="/projects">Projects</a></li>
-			<li class="nav-item" data-active="false"><a onClick="route()" href="/contact">Connect</a></li>
-		</ul>
-	</nav>
-	<main id="content"></main>
-	<footer>
-
-	<div class="flex">
-		<h1>Ready? Let's chat.</h1>
-		<table>
-		<tbody>
-		<tr>
-			<td>info&commat;kalvis.com</td>
-		<!-- </tr>
-			<td>Phone:</td><td> +358 123 123 123</td>
-		</tr> -->
-		</tbody>
-		</table>
-
-		<!-- <button class="contact-button"><a onClick="route()" href="/contact">To the Form</a></button> -->
+					<!-- Site branding -->
+					<div class="shrink-0 mr-4">
+						<!-- Logo -->
+						<a class="block" href="index.html" aria-label="Kalivs">
+							<img class="max-w-none" src="./images/logo-no-text.png" width="38" height="38" alt="Kalvis">
```

Comment:
The logo could maybe be a little larger, maybe 50x50?

Reply:
punt -> let's mess with this in v0.2